src/Kralizek.Extensions.Configuration.AWSSecretsManager/Internal/GetSecretValueRequestContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using Amazon.SecretsManager.Model;
+
+namespace Kralizek.Extensions.Configuration.Internal
+{
+    public class GetSecretValueRequestContext
```

Comment:
How about `SecretValueContext`? Also, please add the xml doc for this type.

Reply:
Changed to `SecretValueContext` and added the documentation.